### ISSUE TITLE
Can't resolve 'p-queue'

### ISSUE THREAD

**Issue by lougreenwood** [ef4/ember-auto-import#388]
I've just updated a dependency ([`p-queue`](https://github.com/sindresorhus/p-queue)) from V6 to V7. V7 of this dependency converts it to an ESM, so I assumed everything should be ok (changelog is here: https://github.com/sindresorhus/p-queue/releases)

However, now the app won't build with the following error:

```
Build Error (Bundler)

Can't resolve 'p-queue' in '/path/to/my-project' 
```

I didn't change any usage of the dependency, I'm importing it as usual: `import PQueue from 'p-queue';`

Not sure what is causing it, here are the logs: 
[error.dump.149fe4dbd1999c1cafadd1f71212f50f.log](https://github.com/ef4/ember-auto-import/files/6367518/error.dump.149fe4dbd1999c1cafadd1f71212f50f.log)

**Comment by ef4** [comment]
It looks like they are relying on the `exports` key in package.json, which means you need webpack 5, which means you need embroider 2.0, which is in beta right now.

**Comment by lougreenwood** [comment]
Fixed by upgrading to `ember-auto-import` 2.0
